packages/common/src/types/semanticLayer.ts

```diff
@@ -47,6 +48,10 @@ export type SemanticLayerField = {
     availableOperators: SemanticLayerStringFilterOperator[];
 };
 
+export type SemanticLayerColumn = VizSqlColumn & {
```

Comment:
I added this type -- would we rather use `PivotChartData`? The name is confusing, but maybe we want a shared type? On the other hand, we have other semantic viewer specific types here.

Reply:
More of a red flag that uses `VizSqlColumn` which sounds like it's sql-runner specific